board/safety/safety_subaru.h

```diff
@@ -20,6 +20,14 @@ const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = {
   .type = TorqueDriverLimited,
 };
 
+const LongitudinalLimits SUBARU_LONG_LIMITS = {
+  .max_brake = 400,  // approx -2.5 m/s^2
+
+  .min_gas = 0,
+  .max_gas = 3400,   // approx  1.2 m/s^2 at low speeds, less at higher speeds
+  .inactive_gas = 808,
+};
+
```

Comment:
We usually allow 2 m/s^2 and -3.5 m/s^2 in safety

Reply:
I'll increase brake limit, but I'm going to leave gas limit as is until I can test max RPM and max gas in our subaru